frontend/app/templates/components/nav-bar.hbs

```diff
@@ -21,7 +21,7 @@
         {{/if}}
```

Comment:
Again, I don't see why this file is even touched. This story is purposefully just for the SSR Frontend.

Reply:
Again Merge Conflict.